modules/swagger-codegen/src/main/resources/typescript-fetch/api.mustache

```diff
@@ -164,7 +164,9 @@ export const {{classname}}FetchParamCreator = function (configuration?: Configur
                 {{/isDateTime}}
                 {{^isDateTime}}
                 {{#isDate}}
-                localVarQueryParameter['{{baseName}}'] = ({{paramName}} as any).toISOString();
+                localVarQueryParameter['{{baseName}}'] = ({{paramName}} as any instanceof Date) ?
+                    ({{paramName}} as any).toISOString().substr(0,10) :
```

Comment:
Just a notice, if the param type is "datetime" the string length should surpass 10 characters since it contains the timestamp as well.
For instance: "2014-01-01T23:28:56.782Z" or even "2014-02-01T09:28:56.321-10:00"

I suggest checking the type for date if you want to keep the substr(0,10) or else this would be problematic for datetime types.